src/asmcnc/core_UI/popups.py

```diff
@@ -1068,4 +1073,78 @@ def reset(*args):
             button_layout_spacing=button_layout_spacing,
             main_label_size_hint_y=main_label_size_hint_y,
             **kwargs
-        )
\ No newline at end of file
+        )
+
+
+class DustshoeWarningPopup(PopupBase):
```

Comment:
Please make the popup a little smaller. Also the OK button please. Feels like they scream at me... :)